process/coordinator/process.go

```diff
@@ -431,6 +438,40 @@ func (tc *transactionCoordinator) CreateMbsAndProcessTransactionsFromMe(maxTxRem
 		}
 	}
 
+	interMBs := tc.processAddedInterimTransactions()
```

Comment:
Why are we doing this two times? ( If we have to choose I like the one liner solution from line 446 better :) )

Reply:
done